lib/irclib/client.js

```diff
@@ -60,6 +66,9 @@ BridgedClient.prototype.setIrcUserInfo = function(ircUser) {
     this.server = ircUser.server;
     this.nick = ircUser.nick;
     this.password = ircUser.password;
+    if (ircUser.localAddress) {
+        this.localAddress = ircUser.localAddress;
+    }
```

Comment:
Not sure, it was in Tor's original patch, so I assumed it was there for a reason, but perhaps not.